### DISCUSSION TITLE
Most Preprocessor's not working, simple "ERROR" message in preview

### DISCUSSION THREAD

**Discussion by emmanceb** [Mikubill/sd-webui-controlnet#934] [Q&A]
The only preprocessors that work as far as I know are invert, lineart_standard, tile_gausan.
Everything else gives me that Error message. 

Only the working preprocessors produce anything through Controlnet. 
All the other selections generate images as if it's not enabled. 

![456](https://user-images.githubusercontent.com/71357653/233439867-c3f05efb-2099-4e6b-9244-2dcb8c83b6be.PNG)

**Comment by COKO-JUNMOQI** [comment]
I am also in this situation, RuntimeError: unexpected EOF, expected 1040992 more bytes. The file might be corrupted.

**Reply by emmanceb** [reply]
Maybe, a reinstalled just fixed it

**Reply by Qq1234a** [reply]
just did, same error

**Comment by Qq1234a** [comment]
I encountered the same problem, but the preprocessor is different. Is it solved?

**Reply by emmanceb** [reply]
Most started working but discovered a few that still wont, not sure why.

**Reply by emmanceb** [reply]
are you on paperspace?
